pkg/test/http_recorder.go

```diff
@@ -18,7 +18,7 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
+	"io"
```

Comment:
nice, thanks for doing that 💯 

reviewer note: ioutil is deprecated; I know there's a linter for that we just haven't enabled it yet.

Reply:
The underlines were bugging me :-)